request.md

```diff
@@ -38,7 +38,7 @@
  * Often the Privacy Notice for your product will link to where the documentation is expected to be.
  * Common examples for Mozilla products/services:
     * If this collection is Telemetry you can state "This collection is documented in its definitions files Histograms.json, Scalars.yaml, and/or Events.yaml and in the Probe Dictionary at https://probes.telemetry.mozilla.org."
-    * If this data is collected using the Glean SDK you can state “This collection is documented in its metrics.md document” and provide a link to where it is hosted.
+    * If this data is collected using the Glean SDK you can state “This collection is documented in its definition files metrics.yaml, pings.yaml and in the Glean Dictionary at https://dictionary.protosaur.dev/"
```

Comment:
I think we can stop mentioning the definitions files (as much as it clearly mirrors the Telemetry case above) as they're not the most human-readable.

```suggestion
    * If this data is collected using the Glean SDK you can state “This collection is documented in the Glean Dictionary at https://dictionary.protosaur.dev/"
```